hbase-common/src/main/java/org/apache/hadoop/hbase/io/crypto/tls/X509Util.java

```diff
@@ -106,15 +106,13 @@ public final class X509Util {
     "hbase.server.netty.tls.supportplaintext";
 
   /**
-   * Set the SSL wrapSize for netty. This is only a maximum wrap size. Buffers smaller
-   * than this will not be consolidated, but buffers larger than this will be split into
-   * multiple wrap buffers. The netty default of 16k is not great for hbase which tends to
-   * return larger payloads than that, meaning most responses end up getting chunked up.
-   * This leads to more memory contention in netty's PoolArena.
-   * See https://github.com/netty/netty/pull/13551
+   * Set the SSL wrapSize for netty. This is only a maximum wrap size. Buffers smaller than this
+   * will not be consolidated, but buffers larger than this will be split into multiple wrap
+   * buffers. The netty default of 16k is not great for hbase which tends to return larger payloads
+   * than that, meaning most responses end up getting chunked up. This leads to more memory
+   * contention in netty's PoolArena. See https://github.com/netty/netty/pull/13551
    */
-  public static final String HBASE_SERVER_NETTY_TLS_WRAP_SIZE =
-    "hbase.server.netty.tsl.wrapSize";
+  public static final String HBASE_SERVER_NETTY_TLS_WRAP_SIZE = "hbase.server.netty.tsl.wrapSize";
```

Comment:
Hi @bbeaudreault, i think you meant hbase.server.netty.**tls**.wrapSize

Reply:
thanks for that catch!